Software/scan_key/AuthenticationSQL.py

```diff
@@ -1,42 +1,42 @@
 from on_off import *
 from ErrorSQL import *
 import os
+import subprocess
 
 def machineAuth(id, cursor):
-    os.system("hostname > tmp")
-    currMachineID = open('tmp', 'r').read()
-    currMachineID = currMachineID.split()
+    currMachineID = subprocess.check_output(['hostname'])
```

Comment:
We still need to strip and decode the byte string.

http://stackoverflow.com/questions/6269765/what-does-the-b-character-do-in-front-of-a-string-literal